src/Process/3dTilesProcessing.js

```diff
@@ -88,6 +100,51 @@ export function pre3dTilesUpdate(context, layer) {
     // const HFOV = 2.0 * Math.atan(Math.tan(radAngle * 0.5) / context.camera.ratio);
     const HYFOV = 2.0 * Math.atan(Math.tan(radAngle * 0.5) * hypotenuse / context.camera.width);
     context.camera.preSSE = hypotenuse * (2.0 * Math.tan(HYFOV * 0.5));
+
+    // once in a while, garbage collect
```

Comment:
If I read correctly, this `once in a while` is actually 98% of the time ?

Reply:
Yep... I'll fix this typo :)

Do you think we can keep this random based version for now, or should we move to another mechanism?

Reply:
Both : we can keep that as a first experiment, but we should think about a garbage-collecting mechanism, possibly unified for every type of layers. 

Some thoughts:
- we should do a full garbage collect when the user does nothing, meaning after we have stopped the update loop (with a 200ms timeout or something similar).
- maybe it's not enough (because if the user keeps moving, we still want to clean from time to time). In this case, we might want to clean regularly, while keeping the load down. Maybe clean only things that have been hidden for a long time, or very far from being visible again? But how to calculate this metrics? Or maybe a time-based metric: we measure previous garbage-collecting process, and depending on the time it took, we define a number of objects/layers/things we can clean at each update without affecting frame rate too much.